### PR TITLE
Small content fixes and style tweaks

### PR DESCRIPTION
**Pull request by ehudtal**

### CODE REVIEW

File: modules-html/4.html [hunk 1]
```diff
@@ -1070,21 +1071,23 @@ <h4 class="box-title">Quick Question</h4>
...
-        Embedded deck not working? Try viewing it <a href="https://www.slideshare.net/thinkoutsidetheslide/ideas-for-effective-standalone-presentations" target="_blank">here</a> instead.<span class="media-duration">57 slides / 4 minutes</span></figcaption>
+      	<span class="media-duration">57 slides / 4 minutes</span>
+        <span class="media-instructions">Embedded slideshow not working? Try viewing it <a href="https://www.slideshare.net/thinkoutsidetheslide/ideas-for-effective-standalone-presentations" target="_blank">here</a> instead.</span>
+        </figcaption>
     </figure>
   </div>
   <div class="bz-box question">
     <h4 class="box-title">Quick Practice</h4>
-    <p>Taking what you learned from this slide deck, sort the following into the correct <em>row</em>.</p>
+    <p>Taking what you learned from this slide , sort the following into the correct <em>row</em>.</p>
```
Comment (sadleb) [inline modules-html/4.html]: Extra space left in there?

File: modules-html/4.html [hunk 2]
```diff
@@ -1070,21 +1071,23 @@ <h4 class="box-title">Quick Question</h4>
...
     <h4 class="box-title">Answer</h4>
     <p>That's right, slide decks are used in virtually every profession. They are a great way to communicate information visually and tell stories.</p>
-    <p>Often, slide decks are used as complements to <strong>in-person</strong> presentations (you've probably given or received a presentation paired with PowerPoint). But just as often, decks must <strong>stand alone</strong>, because there's no opportunity to give a presentation in person, or people prefer to digest the deck on their own.</p>
+    <p>Often, slide decks are used as complements to <strong>in-person</strong> presentations (you've probably given or received a presentation paired with PowerPoint). But just as often, decks must <strong>stand alone</strong>, because there's no opportunity to give a presentation in person, or people prefer to digest the  on their own.</p>
   </div>
   <div class="bz-box video">
     <h4 class="box-title">Click Through This</h4>
     <p>&nbsp;</p>
-    <p>This slide deck gives great advice for how to create a stand-alone slide deck (a bit "meta"!)</p>
+    <p>This slide  gives great advice for how to create a stand-alone slide  (a bit "meta"!)</p>
```
Comment (sadleb) [inline modules-html/4.html]: Extra space left in there?

File: modules-html/4.html [hunk 3]
```diff
@@ -1070,21 +1071,23 @@ <h4 class="box-title">Quick Question</h4>
   <div class="bz-box answer">
     <h4 class="box-title">Answer</h4>
     <p>That's right, slide decks are used in virtually every profession. They are a great way to communicate information visually and tell stories.</p>
-    <p>Often, slide decks are used as complements to <strong>in-person</strong> presentations (you've probably given or received a presentation paired with PowerPoint). But just as often, decks must <strong>stand alone</strong>, because there's no opportunity to give a presentation in person, or people prefer to digest the deck on their own.</p>
+    <p>Often, slide decks are used as complements to <strong>in-person</strong> presentations (you've probably given or received a presentation paired with PowerPoint). But just as often, decks must <strong>stand alone</strong>, because there's no opportunity to give a presentation in person, or people prefer to digest the  on their own.</p>
```
Comment (sadleb) [inline modules-html/4.html]: Extra space left in there?

File: modules-html/4.html (hunk 2 above)
Comment (ehudtal) [reply modules-html/4.html]: wtf? looks like i find&replaced deck with nothing! will fix

**Pull request merged**